### DISCUSSION TITLE
Use elemental as base OS for Rancher RKE2 cluster?

### DISCUSSION THREAD

**Discussion by Sporesirius** [rancher/elemental#1028] [Q&A] [answered]
Hi, 
I'm a bit unsure about where to start with the Rancher ecosystem.
Initially, I would like to experiment with VMware, and perhaps later with Harvester. In the Rancher documentation, the setup involves using K8s and any Linux distribution. I'm curious if this project could also serve as the base OS for a RKE2 Kubernetes cluster to host Rancher, or if there are plans to offer it as an appliance, ISO or similar solution? As I understand it, Elemental is currently used when you already have a K8s cluster setup with Rancher and you want to add more machines.

**Comment by kkaempf** [comment, marked as answer]
Hi Julius,

our primary focus is deployment of new machines/clusters, esp. for edge scenarios.
However, that in no way limits the possibilities of Elemental. It's just our current way to
manage the (always limited) engineering bandwidth.

Of course this project _can_ be used as a base OS for a Rancher (management) cluster.
Harvester is the best example, as it bundles Elemental with Rancher for management.

See #911 - installing Elemental+Rancher is on our backlog ... 😉
